diff -up ppc64-diag-2.6.1/diags/Makefile.verbose ppc64-diag-2.6.1/diags/Makefile
--- ppc64-diag-2.6.1/diags/Makefile.verbose	2013-02-08 10:06:13.000000000 +0100
+++ ppc64-diag-2.6.1/diags/Makefile	2014-03-03 18:25:35.000000000 +0100
@@ -17,11 +17,11 @@ all: $(PROGS)
 
 diag_encl: $(DIAG_ENCL_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) -o $@ $^ $(DIAG_ENCL_LIBS)
+	$(CC) $(CFLAGS) -o $@ $^ $(DIAG_ENCL_LIBS)
 
 encl_led: $(ENCL_LED_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) -o $@ $^
+	$(CC) $(CFLAGS) -o $@ $^
 
 diag_encl.o: encl_util.h diag_encl.h
 
diff -up ppc64-diag-2.6.1/lpd/test/Makefile.verbose ppc64-diag-2.6.1/lpd/test/Makefile
--- ppc64-diag-2.6.1/lpd/test/Makefile.verbose	2014-03-03 18:35:00.000000000 +0100
+++ ppc64-diag-2.6.1/lpd/test/Makefile	2014-03-03 18:35:04.000000000 +0100
@@ -2,7 +2,6 @@
 # Makefile for lp/test
 #
 
-.SILENT:
 include ../../rules.mk
 
 CMDS = ledtool sesdevices
diff -up ppc64-diag-2.6.1/Makefile.verbose ppc64-diag-2.6.1/Makefile
--- ppc64-diag-2.6.1/Makefile.verbose	2013-02-08 10:06:13.000000000 +0100
+++ ppc64-diag-2.6.1/Makefile	2014-03-03 18:25:35.000000000 +0100
@@ -4,8 +4,6 @@
 
 include rules.mk
 
-.SILENT:
-
 SUBDIRS = rtas_errd diags scripts ela lpd
 
 LICENSE = COPYRIGHT
diff -up ppc64-diag-2.6.1/rtas_errd/Makefile.verbose ppc64-diag-2.6.1/rtas_errd/Makefile
--- ppc64-diag-2.6.1/rtas_errd/Makefile.verbose	2013-02-08 10:06:13.000000000 +0100
+++ ppc64-diag-2.6.1/rtas_errd/Makefile	2014-03-03 18:25:35.000000000 +0100
@@ -29,11 +29,11 @@ rtas_errd: $(RTAS_ERRD_OBJS)
 
 convert_dt_node_props: $(DT_NODE_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) -o $@ $^
+	$(CC) $(CFLAGS) -o $@ $^
 
 extract_platdump: $(EXTRACT_PLATDUMP_OBJS)
 	@echo "LD $(WORK_DIR)/$@"
-	@$(CC) $(CFLAGS) -o $@ $^ $(EXTRACT_PLATDUMP_LIBS)
+	$(CC) $(CFLAGS) -o $@ $^ $(EXTRACT_PLATDUMP_LIBS)
 
 install: all
 	@$(call install_sbin,$(CMDS),$(DESTDIR))	
diff -up ppc64-diag-2.6.1/rules.mk.verbose ppc64-diag-2.6.1/rules.mk
--- ppc64-diag-2.6.1/rules.mk.verbose	2013-02-08 10:06:13.000000000 +0100
+++ ppc64-diag-2.6.1/rules.mk	2014-03-03 18:25:35.000000000 +0100
@@ -199,7 +199,6 @@ CLEAN = @echo ""
 
 # Default target for building object files
 %.o: %.c
-	@echo "CC $(WORK_DIR)/$@"
-	@$(CC) -c $(AM_CFLAGS) $(CFLAGS) $<
+	$(CC) -c $(AM_CFLAGS) $(CFLAGS) $<
 
 
